src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeMapperTest.java

```diff
@@ -79,10 +114,27 @@ private void assertSameScannableItems(Envelope dbEnvelope, InputEnvelope zipEnve
 
         assertThat(dbEnvelope.getScannableItems())
             .extracting(this::convertToInputScannableItem)
-            .usingRecursiveFieldByFieldElementComparator()
+            .usingElementComparatorIgnoringFields("ocrData", "documentType")
             .containsAll(zipEnvelope.scannableItems);
     }
 
+    private void assertSameOcrData(Envelope dbEnvelope, InputEnvelope zipEnvelope) {
+        assertThat(dbEnvelope.getScannableItems())
+            .extracting(ScannableItem::getOcrData)
+            .extracting(this::convertToInputOcrData)
+            .usingRecursiveFieldByFieldElementComparator()
+            .containsAll(
+                zipEnvelope.scannableItems.stream().map(item -> item.ocrData).collect(toList())
+            );
+    }
+
+    private void assertDocumentTypes(Envelope dbEnvelope, DocumentType... documentTypes) {
+        assertThat(dbEnvelope.getScannableItems())
+            .extracting(ScannableItem::getDocumentType)
+            .usingDefaultComparator()
+            .containsExactly(documentTypes);
```

Comment:
We need to make sure order is preserved.

Reply:
This method containsExactly preserves order unlike containsExactlyInAnyOrder

Reply:
aaa, true